torch/csrc/jit/frontend/schema_matching.cpp

```diff
@@ -324,16 +324,23 @@ static bool varargsCanBeUsedAsList(
       !typevar_list;
 }
 
-// Note (@zasdfgbnm):
-// This is a workaround for https://github.com/pytorch/pytorch/issues/47964
-// Currently JIT does not distinguish ScalarType vs int, so there is really
-// no way to distinguish x.view(1) vs x.view(torch.int8). So we have to hardcode
-// the aten::view.dtype here to block this overload. This blocklist should be
-// removed when JIT fully suports ScalarType as its own type.
 bool isBlockListedSchema(const FunctionSchema& schema) {
+  // Note (@zasdfgbnm):
+  // This is a workaround for https://github.com/pytorch/pytorch/issues/47964
+  // Currently JIT does not distinguish ScalarType vs int, so there is really
+  // no way to distinguish x.view(1) vs x.view(torch.int8). So we have to
+  // hardcode the aten::view.dtype here to block this overload. This blocklist
+  // should be removed when JIT fully suports ScalarType as its own type.
   if (schema.name() == "aten::view" && schema.overload_name() == "dtype") {
     return true;
   }
+  // Note (@tugsbayasgalan)
+  // TorchScript doesn't suport kwargs so this op collides with aten.max.others
+  // since both of them have 2 Tensor inputs. Since we don't expect users to
+  // use this op in TS, we just skip it
+  if (schema.name() == "aten::max" && schema.overload_name() == "unary_out") {
+    return true;
```

Comment:
hmm what's the consequence of this - I guess one consequence is that if you call `torch.ops.aten.max(tensor, tensor)`, we'll continue using `max.out` instead of `max.unary_out` (which seems reasonable). Anything else?

Reply:
TS incorrectly resolves this operator to aten::max.other. If we do this, TS will complain about how there is no matching signature, hence will error out. This is fine because we don't expect TS users to use this op anyways and the proper fix for this is gonna be pretty involved.